### ISSUE TITLE
Faster gradient computation

### ISSUE THREAD

**Issue by nicolasugrinovic** [polmorenoc/opendr#3]
Thank you for releasing this! 
OpenDR uses chumpy for gradient computations which (if I am not wrong) run on CPU, is there a way to accelerate this? Have you tried to move this to Tensorflow, PyTorch or some other framework?

**Comment by ahnHeejune** [comment]
Yes, I thought the same thing as above.  Do you think we can use tensorflow instead of chumpy?  Both providing similar functions
